sched/sched/sched_lock.c

```diff
@@ -64,112 +83,25 @@
  *
  ****************************************************************************/
 
-#ifdef CONFIG_SMP
-
-int sched_lock(void)
+void sched_lock(void)
 {
-  FAR struct tcb_s *rtcb;
-
-  /* If the CPU supports suppression of interprocessor interrupts, then
-   * simple disabling interrupts will provide sufficient protection for
-   * the following operation.
-   */
-
-  rtcb = this_task();
+#if (CONFIG_SCHED_CRITMONITOR_MAXTIME_PREEMPTION >= 0) ||\
+    defined(CONFIG_SCHED_INSTRUMENTATION_PREEMPTION)
+  irqstate_t flags = enter_critical_section();
```

Comment:
```suggestion
  irqstate_t flags = enter_critical_section_wo_note();
```
should be `enter_critical_section_wo_note()` ?

Reply:
done